addons/web/static/src/views/fields/image/image_field.js

```diff
@@ -43,9 +44,34 @@ export class ImageField extends Component {
             isValid: true,
         });
         this.lastURL = undefined;
+
+        if (this.props.record.fields[this.props.name].related) {
+            this.lastUpdate = this._now();
+            const cacheKey = () => ({
+                value: this.props.value,
+                resId: this.props.record.resId,
+            });
+            this.key = cacheKey();
+            onWillRender(() => {
+                const nextKey = cacheKey();
+
+                if (!shallowEqual(this.key, nextKey)) {
+                    this.lastUpdate = this._now();
+                }
+
+                this.key = nextKey;
+            });
+        }
+    }
+
+    _now() {
+        return DateTime.now();
```

Comment:
nice objection by @aab-odoo 
imagine a kanban where every record has the same image (related to the same record), the browser will fetch
as many times the image as there are records

Reply:
Isn't that already the case since the id in the queryParam is the resId of the main record ?